spec/support/jruby_binstubs/bundle

```diff
@@ -0,0 +1,25 @@
+require "rubygems"
+gem "bundler"
+require "bundler"
+require "bundler/fetcher"
+require_relative "../in_process_exec.rb"
+
+module Bundler
+  #:nodoc:
+  class Fetcher
+    # The Bundler user_agent uses SecureRandom, which causes the specs
+    # to run out of entropy and run a lot longer than they need to.
+    def user_agent
+      "gemstash spec"
+    end
+  end
+end
```

Comment:
I wonder if it makes sense to create a Random generator based on a SecureRandom seed and just keep it around.

No a blocker, though